comms/DiskRepository.cpp

```diff
@@ -138,6 +138,10 @@ class CDiskRepository : public CRepositoryBase
                 pos = moduleName.rfind(_T("\\"));
                 moduleName = moduleName.substr(0, pos);
                 algo::replace_all(moduleName, _T("\\"), _T("."));
+                if (!module.empty() && !boost::equal(moduleName, module) && !boost::starts_with(moduleName, module + _T(".")))
```

Comment:
What is the "starts_with" used for?

Reply:
@GordonSmith it is used for making sure we math only Test with "Test." and not "Tester."

Reply:
ok